bentoml/yatai/yatai_service.py

```diff
@@ -48,22 +53,22 @@ def get_yatai_service(
         header_adder_interceptor = header_client_interceptor.header_adder_interceptor(
             access_token_header, access_token
         )
-        if scheme in ('grpcs', 'https'):
+        if scheme in ("grpcs", "https"):
```

Comment:
See above

Reply:
I just notice that after changing back to the default black automatically change all single quote to double quote. I was wondering whether you got the same output if you run `./dev/format.sh`